src/extractors/jsxtag-gettext.js

```diff
@@ -1,10 +1,57 @@
 import * as t from 'babel-types';
 import gettext from './tag-gettext';
+import { PO_PRIMITIVES } from '../defaults';
+import { template2Msgid, msgid2Orig, hasExpressions,
+    ast2Str, getQuasiStr, strToQuasi, dedentStr } from '../utils';
+import { NoTranslationError } from '../errors';
 
+const { MSGSTR } = PO_PRIMITIVES;
 const NAME = 'jsxtag-gettext';
 
 function match(node, context) {
     return t.isTaggedTemplateExpression(node) && node.tag.name === context.getAliasFor(NAME);
 }
 
-export default { ...gettext, match, name: NAME };
+function resolveDefault(nodePath, poData, context) {
```

Comment:
For `resolveDefault()` and `resolve()` here,
I was thinking that I would need to implement these that resolves input strings to an array.
However, after putting `console.log` in them I found that neither of them is invoked when `tests/functional/test_resolve_jsxtag_gettext.js` is running. Then I have no idea how to continue 😕